notebooks/official/custom/custom_training_tensorboard_profiler.ipynb

```diff
@@ -0,0 +1,1639 @@
+{
```

Comment:
<p>TensorFlow versions older than 2.3 will fail to load the custom model in this tutorial if a GPU is specified?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/719/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='50'/>

Reply:
<p>This notebook requires TensorFlow 2.4 to run and it will automatically update the TensorFlow version. And it also use TensorFlow 2.8 and up in the container images. So the above note about TensorFlow 2.3 and oder versions is not longer relevant. I am deleting it. Thanks.</p>